snippets/functions/third-party/manifest.json

```diff
@@ -1,7 +1,13 @@
 {
-    "category": "third-party",
-    "categoryId": "51a6be3b-a3f2-427c-937f-f810a91dd78c",
-    "viewType": "functionSnippet",
-    "snippets": [
-    ]
-}
\ No newline at end of file
+  "category": "third-party",
+  "categoryId": "51a6be3b-a3f2-427c-937f-f810a91dd78c",
+  "viewType": "functionSnippet",
+  "snippets": [
+    {
+      "id": "e5450427-8503-4e1a-aac3-03e5e39e8f8e",
+      "title": "Calling Twilio service via NPM dependency",
```

Comment:
@laurazhukas Can you check this third party snippet's title and description to see if it is okay?